src/transaction/boot_cl.c

```diff
@@ -5735,14 +6321,34 @@ boot_define_view_db_collation (void)
 	}
     }
 
+  // *INDENT-OFF*
   sprintf (stmt,
-	   "SELECT [c].[coll_id], [c].[coll_name], [ch].[charset_name], CASE [c].[built_in] WHEN 0 THEN 'No'"
-	   " WHEN 1 THEN 'Yes' ELSE 'ERROR' END, CASE [c].[expansions] WHEN 0 THEN 'No'"
-	   " WHEN 1 THEN 'Yes' ELSE 'ERROR' END, [c].[contractions], CASE [c].[uca_strength]"
-	   " WHEN 0 THEN 'Not applicable' WHEN 1 THEN 'Primary' WHEN 2 THEN 'Secondary'"
-	   " WHEN 3 THEN 'Tertiary' WHEN 4 THEN 'Quaternary' WHEN 5 THEN 'Identity' ELSE 'Unknown' END"
-	   " FROM [%s] [c] JOIN [%s] [ch] ON [c].[charset_id] = [ch].[charset_id] ORDER BY [c].[coll_id]",
-	   CT_COLLATION_NAME, CT_CHARSET_NAME);
+	"SELECT "
+	  "[coll].[coll_id] AS [coll_id], "
+	  "[coll].[coll_name] AS [coll_name], "
+	  "[ch].[charset_name] AS [charset_name], "
+	  "CASE [coll].[built_in] WHEN 0 THEN 'No' WHEN 1 THEN 'Yes' ELSE 'ERROR' END AS [is_builtin], "
+	  "CASE [coll].[expansions] WHEN 0 THEN 'No' WHEN 1 THEN 'Yes' ELSE 'ERROR' END AS [has_expansions], "
```

Comment:
it looks need to re-arange for CASE caluses.

Reply:
If there are not many branching values and does not exceed 120 bytes, it is written in one line.
If there is no other opinion, I will keep it.